### PR TITLE
Changed to conform to style guide

### PR DESCRIPTION
**Pull request by theundeadmonk**

### CODE REVIEW

**Comment by ashtonthomas** [comment]
tabz. tabz r bad. tabz-to-spaces ;-)

See if you can configure your editor to convert tabs to spaces (2spaces per tab is the recommended setting)

File: triangle_facts/triangle_facts.rb [hunk 1]
```diff
@@ -7,28 +7,32 @@ def initialize(side1,side2,side3)
...
-	    true
-	  end
+        if equilateral || isosceles
+	        false
+	    else
+	        true
+        end
 	end
 
 	def recite_facts
```
Comment (ashtonthomas) [inline triangle_facts/triangle_facts.rb]: This method mixes tabz and spaces

**Comment by ashtonthomas** [comment]
I can't comment inline because your diff doesn't touch it but:
can we get some spaces between these vars? `attr_accessor :side1,:side2,:side3`

**Comment by ashtonthomas** [comment]
multi line assignment is frowned upon :-(

`@side1,@side2,@side3 = side1,side2,side3`

File: triangle_facts/triangle_facts.rb [hunk 2]
```diff
@@ -7,28 +7,32 @@ def initialize(side1,side2,side3)
 	end
 
 	def equilateral()
```
Comment (ashtonthomas) [inline triangle_facts/triangle_facts.rb]: No need to parens here or anywhere else you don't have parameters

File: triangle_facts/triangle_facts.rb [hunk 3]
```diff
@@ -7,28 +7,32 @@ def initialize(side1,side2,side3)
...
-		return side1 == side2 && side2 == side3
+		side1 == side2 && side2 == side3
 	end
 
 	def isosceles()
-		return [side1,side2,side3].uniq.length == 2
+		[side1,side2,side3].uniq.length == 2
```
Comment (ashtonthomas) [inline triangle_facts/triangle_facts.rb]: some spaces may clean this up a bit. Having everything compact can make it hard to read

File: triangle_facts/triangle_facts.rb [hunk 4]
```diff
@@ -7,28 +7,32 @@ def initialize(side1,side2,side3)
...
 	end
 
 	def scalene()
-		unless ! (equilateral || isosceles)
-	    false
-	  else
-	    true
-	  end
+        if equilateral || isosceles
```
Comment (ashtonthomas) [inline triangle_facts/triangle_facts.rb]: We can probably write this with less lines ;-)

**Comment by ashtonthomas** [comment]
maybe some spaces here: `def calculate_angles(a,b,c)`

File: triangle_facts/triangle_facts.rb [hunk 5]
```diff
@@ -7,28 +7,32 @@ def initialize(side1,side2,side3)
...
+	    else
+	        true
+        end
 	end
 
 	def recite_facts
-		puts 'This triangle is equilateral!' if equilateral
-		puts 'This triangle is isosceles! Also, that word is hard to type.' if isosceles 
-		puts 'This triangle is scalene and mathematically boring.' if scalene 
+        if equilateral
```
Comment (ashtonthomas) [inline triangle_facts/triangle_facts.rb]: not sure the if/else is an improvement from having the conditionals at the end

**Comment by ashtonthomas** [comment]
@theundeadmonk, I provided some comments that should help you to continue the refactor. Want to take a look through and submit an update PR?

File: triangle_facts/triangle_facts.rb (hunk 5 above)
Comment (theundeadmonk) [inline triangle_facts/triangle_facts.rb]: Having the if else block will ensure faster execution since all the conditionals need not be evaluated.

**Pull request closed without merging**